Gemfile

```diff
@@ -11,6 +11,7 @@ end
 
 # #2
 group :development do
+  gem 'factory_bot_rails'
```

Comment:
This is used for testing.  So you'll want it in a testing group, or dev/test group.  i.e. https://github.com/nagano564/house/blob/5da40c0ad18c4dfeff05b2a511caf463bd32cd2d/Gemfile#L44-L48